homeassistant/components/p2pcam/const.py

```diff
@@ -0,0 +1,3 @@
+"""Constants for the p2pcam component."""
+
+DEFAULT_NAME = "p2pcam"
```

Comment:
If you only have one constant move it to the camera.py file and delete this one.